src/OrchardCore/OrchardCore.Logging.NLog/WebHostBuilderExtensions.cs

```diff
@@ -30,10 +31,12 @@ internal static class AspNetExtensions
     {
         public static LoggingConfiguration ConfigureNLog(this IHostEnvironment env, string configFileRelativePath)
         {
-            ConfigurationItemFactory.Default.RegisterItemsFromAssembly(typeof(AspNetExtensions).GetTypeInfo().Assembly);
```

Comment:
Instead of depending on assembly-scanning, then it could nice if just doing registration of the single custom LayoutRenderer:
```c#
var fileName = Path.Combine(env.ContentRootPath, configFileRelativePath);
LogManager.Setup().SetupLogFactory(factory => factory.AddCallSiteHiddenAssembly(typeof(AspNetExtensions).GetTypeInfo().Assembly))
          .SetupExtensions(ext => {
                ext.RegisterLayoutRenderer<TenantLayoutRenderer>(TenantLayoutRenderer.LayoutRendererName);
                ext.RegisterNLogWeb();
          })
          .LoadConfigurationFromFile(fileName);
return LogManager.Configuration;
```